packages/api/lib/rulesHelpers.js

```diff
@@ -589,6 +589,7 @@ async function invokeRerun(rule) {
   await invoke(process.env.invoke, payload);
 }
 
+/*eslint complexity: ["error", 16]*/
```

Comment:
This should not be commented out.   Generally, we're exceeding the cyclo complexity threshold, we need to refactor the method, and add tests accordingly.

Reply:
We definitely don't want to case-modify the cyclo rules here.    If we're pushing it over the codebase limit we should refactor/abstract it without good technical reasons not to. 

With regard to the rule behavior, I think your second comment makes sense, but you could certainly ask for more feedback.